packages/workflow/src/Interfaces.ts

```diff
@@ -1535,3 +1535,19 @@ export interface IConnectedNode {
 	indicies: number[];
 	depth: number;
 }
+
+export enum OAuth2GranType {
```

Comment:
Minor type, this should probably be `OAuth2GrantType`

Reply:
Ahh yes, good catch.